Gemfile

```diff
@@ -67,6 +67,7 @@ gem "omniauth", "~> 1.9" # A generalized Rack framework for multiple-provider au
 gem "omniauth-github", "~> 1.3" # OmniAuth strategy for GitHub
 gem "omniauth-twitter", "~> 1.4" # OmniAuth strategy for Twitter
 gem "patron", "~> 0.13.3" # HTTP client library based on libcurl, used with Elasticsearch to support http keep-alive connections
+gem "percy-capybara", "~> 4.0.0"
```

Comment:
I think this should be moved down to the `:test` group

Reply:
ps. don't we need the percy agent as well? https://docs.percy.io/docs/capybara#setup

Reply:
Good call on the `:test` group! I honestly don't know how this worked _without_ me adding the percy agent...? Maybe because I was running it locally 🤔 But yes, you are right that I need it and I have no idea why it worked but OH WELL, I have added it!